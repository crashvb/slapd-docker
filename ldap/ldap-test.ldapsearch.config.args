simple_bind, anonymous=-x
simple_bind, anonymous, TLS requested=-x -Z
simple_bind, anonymous, TLS forced=-x -ZZ
simple_bind, unauthenticated=-x -D 'cn=admin,cn=config'
simple_bind, unauthenticated, TLS requested=-x -D 'cn=admin,cn=config' -Z
simple_bind, unauthenticated, TLS forced=-x -D 'cn=admin,cn=config' -ZZ
simple_bind, authenticated=-x -D 'cn=admin,cn=config' -w $LDAP_ADMIN_PASSWORD
simple_bind, authenticated, TLS requested=-x -D 'cn=admin,cn=config' -w $LDAP_ADMIN_PASSWORD -Z
simple_bind, authenticated, TLS forced=-x -D 'cn=admin,cn=config' -w $LDAP_ADMIN_PASSWORD -ZZ
SASL=-Y EXTERNAL
SASL, TLS requested=-Y EXTERNAL -Z
SASL, TLS forced=-Y EXTERNAL -ZZ
