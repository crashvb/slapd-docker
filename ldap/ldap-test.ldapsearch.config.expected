ldap, simple_bind, anonymous=disallowed
ldap, simple_bind, anonymous, TLS requested=disallowed
ldap, simple_bind, anonymous, TLS forced=disallowed
ldap, simple_bind, unauthenticated=disallowed
ldap, simple_bind, unauthenticated, TLS requested=disallowed
ldap, simple_bind, unauthenticated, TLS forced=disallowed
ldap, simple_bind, authenticated=confidentiality required
ldap, simple_bind, authenticated, TLS requested=Invalid credentials
ldap, simple_bind, authenticated, TLS forced=Invalid credentials
ldap, SASL=no mechanism available
ldap, SASL, TLS requested=No such object
ldap, SASL, TLS forced=No such object
ldapi, simple_bind, anonymous=disallowed
ldapi, simple_bind, anonymous, TLS requested=disallowed
ldapi, simple_bind, anonymous, TLS forced=disallowed
ldapi, simple_bind, unauthenticated=disallowed
ldapi, simple_bind, unauthenticated, TLS requested=disallowed
ldapi, simple_bind, unauthenticated, TLS forced=disallowed
# Too Restrictive: ldapi, simple_bind, authenticated=confidentiality required
ldapi, simple_bind, authenticated=Invalid credentials
ldapi, simple_bind, authenticated, TLS requested=Invalid credentials
ldapi, simple_bind, authenticated, TLS forced=Invalid credentials
# Too Restrictive: ldapi, SASL=confidentiality required
ldapi, SASL, TLS requested=No such object
ldapi, SASL, TLS forced=No such object
ldaps, simple_bind, anonymous=disallowed
ldaps, simple_bind, anonymous, TLS requested=already started
ldaps, simple_bind, anonymous, TLS forced=already started
ldaps, simple_bind, unauthenticated=disallowed
ldaps, simple_bind, unauthenticated, TLS requested=already started
ldaps, simple_bind, unauthenticated, TLS forced=already started
ldaps, simple_bind, authenticated=Invalid credentials
ldaps, simple_bind, authenticated, TLS requested=Invalid credentials
ldaps, simple_bind, authenticated, TLS forced=already started
ldaps, SASL=No such object
ldaps, SASL, TLS requested=already started
ldaps, SASL, TLS forced=already started
